README.md

```diff
@@ -8,6 +8,17 @@ for a rendered version of this material,
 for instructions on formatting, building, and submitting material,
 or run `make` in this directory for a list of helpful commands.
 
+## Run Jekyll in a Docker container
+
+If you download and install the latest versions of [Docker compose](https://docs.docker.com/compose/install/) and
+[Docker engine](https://docs.docker.com/engine/installation/) you can build a Jekyll container to serve this site with
+hot-reloading and incremental rebuilds by running `docker-compose up` and visiting `localhost:4000` in a browser. 
```

Comment:
I appreciate this addition.  Where specifically should a user run "docker-compose up".  Is thisa Jupyter command, shell command, flie a user should click, etc?